WordPress/gutenberg
# Fix `WordPress.NamingConventions.PrefixAllGlobals` type linter errors

This PR aims to make Gutenberg compatible with the `WordPress-Extra` ruleset by fixing some of the `WordPress.NamingConventions.PrefixAllGlobals` linter errors (from that ruleset).

Fixes https://github.com/WordPress/gutenberg/issues/45817.



## What?
This PR adds a couple of phpcs:ignore rules.

## Why?
This is needed to ensure that Gutenberg is compatible with the `WordPress.NamingConventions.PrefixAllGlobals` `phpcs` rule.

## How?


## Testing Instructions
This PR can be appropriately tested after the `WordPress-Extra` ruleset is enabled.
Please see: https://github.com/WordPress/gutenberg/issues/18502
For now, please ensure that it doesn't generate any phpcs linter errors (it can be verified by utilizing GitHub's CI/CD checks below). Just make sure that there are no failed checks.

## Screenshots or screencast
